charts/coturn/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
```

Comment:
which kubernetes versions does this work with, and did you test the helm chart on? (we have various versions in use, customers might have various versions in use)

Reply:
I've tested with Kubernetes version 1.19.7. I've got no idea what the range of compatible versions is, this was taken from the chart for restund.